src/cockpit/389-console/src/lib/ldap_editor/treeView.jsx

```diff
@@ -113,6 +113,10 @@ class EditorTreeView extends React.Component {
 
         this.handleNodeOnClick = (treeViewItem) => {
             console.log('handleNodeOnClick - treeViewItem: ', treeViewItem);
+            if (treeViewItem.dn === this.state.entryDn) {
```

Comment:
Weirdly enough, the browser crashes here when I:

1. Go to the Tree view and select a user under ou=People;
2. Go to the Table view and select the same user there;
3. Go back to the Tree view and open the same user view (may be an unnecessary step)
4. Go back to the Table view and delete the user.

Result - Browser crashed.

Reply:
I can not reproduce the crash

Reply:
fixed